python-avd/pyavd/_eos_cli_config_gen/j2templates/eos-intended-config.j2

```diff
@@ -229,6 +229,7 @@
 {% include 'eos/ipv6-hardware.j2' %}
 {# Monitor Sessions #}
 {% include 'eos/monitor-sessions.j2' %}
+{% include 'eos/monitor_sessions_default_encapsulation_gre.j2' %}
```

Comment:
we should have hyphen instead of underscore in template name for consistency

Reply:
Fixed